tests/countries/test_ivory_coast.py

```diff
@@ -0,0 +1,266 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from unittest import TestCase
+
+from holidays.countries import IvoryCoast, CI, CIV
+from tests.common import CommonCountryTests
+
+
+class TestIvoryCoast(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1997, 2050)
+        super().setUpClass(IvoryCoast, years=years, years_non_observed=years)
+
+    def test_country_aliases(self):
+        self.assertAliases(IvoryCoast, CI, CIV)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(IvoryCoast(years=1996))
+
+    def test_special_holidays(self):
+        self.assertHoliday(
+            "2010-10-29",
+            "2024-02-12",
+        )
+
+    def test_new_years_day(self):
+        self.assertHolidayName("1er janvier", (f"{year}-01-01" for year in range(1997, 2050)))
+
+    def test_labor_day(self):
+        name = "Fête du travail"
+        self.assertHolidayName(name, (f"{year}-05-01" for year in range(1997, 2050)))
+        dt = (
+            "2005-05-02",
+            "2011-05-02",
+            "2016-05-02",
+        )
+        self.assertHolidayName(f"Lendemain de la {name}", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_independence_day(self):
+        self.assertHolidayName("Fête Nationale", (f"{year}-08-07" for year in range(1997, 2050)))
+        self.assertHolidayName("Lendemain de la Fête Nationale", "2022-08-08")
```

Comment:
To be in line with other test cases:
```suggestion
    def test_independence_day(self):
        name = "Fête Nationale"
        self.assertHolidayName(name, (f"{year}-08-07" for year in range(1997, 2050)))
        dt = ("2022-08-08",)
        self.assertHolidayName(f"Lendemain de la {name}", dt)
        self.assertNoNonObservedHoliday(dt)
```

Reply:
Ignore the approve change message above, I misclicked 😔